contrib/gitian-descriptors/gitian-osx.yml

```diff
@@ -115,9 +115,7 @@ script: |
     cp ${BASEPREFIX}/${i}/native/bin/${i}-codesign_allocate unsigned-app-${i}/codesign_allocate
     cp ${BASEPREFIX}/${i}/native/bin/${i}-pagestuff unsigned-app-${i}/pagestuff
     mv dist unsigned-app-${i}
-    pushd unsigned-app-${i}
-    find . | sort | tar --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ${OUTDIR}/${DISTNAME}-osx-unsigned.tar.gz
-    popd
+    find unsigned-app-${i} | sort | tar --no-recursion --mode='u+rw,go+r-w,a+X' --owner=0 --group=0 -c -T - | gzip -9n > ${OUTDIR}/${DISTNAME}-osx-unsigned.tar.gz
```

Comment:
No, see the PR comment. sig creation is all relative to the working dir, and I've been meaning to make it extract to a subdir rather than polluting the current dir anyway. sig application is impartial, as it's also relative to the working dir.